### ISSUE TITLE
[BUG] Insufficient Color Contrast

### ISSUE THREAD

**Issue by mgifford** [gc-da11yn/gc-da11yn.github.io#205]
The form - Targeted Engagement on the draft Standard on ICT Accessibility (Phase 1)
https://forms.office.com/Pages/ResponsePage.aspx?id=EN-XY5VFR0CcTwMxEoIVK5_R18YD5b5JpOR8T_7y9oxUOUlVWVY5TVlBUDlJU0VPRlNLV0ZINlhDSCQlQCN0PWcu

Doesn't have color contrast:

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.select-placeholder-text)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Targeted Engagement on the draft Standard on ICT Accessibility (Phase 1) - https://forms.office.com/Pages/ResponsePage.aspx?id=EN-XY5VFR0CcTwMxEoIVK5_R18YD5b5JpOR8T_7y9oxUOUlVWVY5TVlBUDlJU0VPRlNLV0ZINlhDSCQlQCN0PWcu

Element path: .select-placeholder-text

Snippet: <span class="select-placeholder-text">English (United States)‎</span>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.25 (foreground color: #ffffff, background color: #1d7fc9, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 106.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.35.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
